src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -129,7 +133,7 @@ def ok_handler
       UI.ReplaceWidget(Id(:busy), Label(_('Configuring instance TLS ...')))
       # Turn on TLS
       if !DS389.install_tls_in_nss(instance_name, tls_ca, tls_p12)
-        Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+        Popup.Error(_('Failed to set up new instance! Log output may be found in %{y2log}'), y2log: y2log)
```

Comment:
Missing [`format`](https://ruby-doc.org/core-3.1.2/Kernel.html#method-i-format)

```suggestion
        Popup.Error(format(_('Failed to set up new instance! Log output may be found in %{y2log}'), y2log: y2log))
```